test/parsing/errors/invalidIndexExpr.chpl

```diff
@@ -0,0 +1,3 @@
+for i.x in [1..5] do {
```

Comment:
Hi @riftEmber — With my new warning for array literals with a single element and no trailing comma, I'm getting two errors for this test now.  Would it be OK if I were to change `[1..5]` to `1..5` to keep this test focused on the invalid index expression?  (I could also change it to `[1..5, ]` if you preferred, that just seemed like a less likely thing for a real user to write to me).